code/modules/power/port_gen.dm

```diff
@@ -94,6 +94,8 @@
 	var/sheet_left = 0 // How much is left of the sheet
 	var/time_per_sheet = 260
 	var/current_heat = 0
+	var/pollution_multiplier = 0
+	var/pollution_gas = GAS_CO
```

Comment:
This stuff feels like it would work better if not for the fact you are inside a ship all the time, moving a generator outside would be complicated and/or tedious and most ships rely on them

Reply:
Yeah it's effectively disabled right now, atmos during testing and on live looked very different. I might keep it then enable it far far later when we got better primary engines

Reply:
iirc Rylies ripping this out for now, for a separate pr.